modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -1,78 +1,118 @@
-= Stop, Restart, or Update an Anypoint Private Cloud Edition Node
+= Update Nodes
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+[IMPORTANT]
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
 
-Before any maintenance or restart of a node can be performed, it must be removed from the cluster and its workload reassigned to the other nodes in the cluster.
+=== Verify Cluster Health Status
 
-. Check if the firewalld service is running and enabled:
+Verify that the cluster is healthy before starting the process. Otherwise, many of the assumptions made in this process are not valid.
+
+. Verify the output from the following commands:
++
+```
+gravity status
+etcdctl cluster-health
+```
++
+If the output from the previous commands mentions a `degraded` state or `errors`, fix all issues before proceeding.
+
+. Verify that all pods are running:
 +
-----
-service firewalld status
-----
-. If firewalld is running, stop, disable, and mask it:
+```
+gravity get pod -o wide--all-namespaces | grep -vi Running
+```
 +
-----
-sudo systemctl stop firewalld
-sudo systemctl disable firewalld
-sudo systemctl mask firewalld
-----
-. Retrieve the *name* of target node:
+If the output from the previous command lists any pod not in "Running" state, then fix all issues before proceeding.
+
+=== Install Updates
+
+After verifying that the cluster is healthy, select a node on which to perform the following steps:
```

Comment:
This intro is not really helpful, because it tells the audience to select a node but gives no criteria other than that they are going to perform installation on it.
I think we might want something more like "After you verify the health of the cluster, pick any node on which to install updates. Remember that you must not only complete this installation but also verify it on this one node before you proceed to update and verify on the next node."